Assets/Scripts/TimeManager.cs

```diff
@@ -16,6 +16,7 @@ public class TimeManager : MonoBehaviour
     private float current_time;
     private float time_elapsed;
     private float velocity_magnitude;
+    // second round of implementation\\
 
     private enum stateMachine { normalSpeed, slowDown, slowMotion, speedUp };
```

Comment:
Can we refactor this so there isn't a separate state machine in this script and `KatamariMovement`? It seems fragile to have two separate ones that could get out of sync with each other.

Reply:
Removed in the follow on PR